### DISCUSSION TITLE
Hide Templates, Theme Builder, and Edit Header/Footer by Role

### DISCUSSION THREAD

**Discussion by portrayaloflife** [elementor/elementor#22319] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Editors or users with edit only roles in Elementor are able to edit the theme builder, view page templates etc, which we do not want for our clients. 

Plus they don't have the ability to actually really add to those anyway since they have edit only access. Would be easier if we can just hide those from them entirely.

### Describe the solution you'd like

The ability to hide Templates, Theme Builder, and Edit Header/Footer by user role. Previosuly if say "Editors" were edit access only they couldn't edit these things but now it seems that they have access to all of those. 

We don't want clients editing page templates or getting into saved templates we had internally that now they can just stumble upon.

### Describe alternatives you've considered

custom functions and CSS

**Comment by marks7777** [comment]
definitely needed feature! I lost content on a few pages because editors are able to edit templates, footers and headers built with the Theme Builder. I would also like to see a more detailed way to set user roles, not just two options.
